src/actor/character/sheets-tabs/main-tab.hbs

```diff
@@ -1,92 +1,99 @@
         </div>
         <div>
           <input type="checkbox" id="dazed" data-dtype="Boolean" name="system.condition.dazed.value" {{checked this.actor.conditions.dazed.value}} />
-          <label for="dazed">Dazed</label>
+          <label for="dazed">{{localize "STATUS_EFFECT.DAZED"}}</label>
         </div>
       </div>
       <div class="attribute-stats might">
-        <p class="attribute">Might</p>
+        <p class="attribute">{{localize "ATTRIBUTE.MIGHT"}}</p>
         <p id="calculated-attribute-value"> d{{this.actor.might}}</p>
         <i class="fas fa-slash rotated"></i>
         <div>
           <input id="number-input" type="number" step=2 min=6 max=12 name="system.baseAttribute.baseMight.value" value="{{this.actor.attributes.baseMight.value}}" />
         </div>
         <div>
           <input type="checkbox" id="weak" data-dtype="Boolean" name="system.condition.weak.value" {{checked this.actor.conditions.weak.value}} />
-          <label for="weak">Weak</label>
+          <label for="weak">{{localize "STATUS_EFFECT.WEAK"}}</label>
         </div>
       </div>
       <div class="attribute-stats willpower">
-        <p class="attribute">Willpower</p>
+        <p class="attribute">{{localize "ATTRIBUTE.WILLPOWER"}}</p>
         <p id="calculated-attribute-value"> d{{this.actor.willpower}} </p>
         <i class="fas fa-slash rotated"></i>
         <div>
           <input id="number-input" type="number" step=2 min=6 max=12 name="system.baseAttribute.baseWillpower.value" value="{{this.actor.attributes.baseWillpower.value}}" />
         </div>
         <div>
           <input type="checkbox" id="shaken" data-dtype="Boolean" name="system.condition.shaken.value" {{checked this.actor.conditions.shaken.value}} />
-          <label for="shaken"> Shaken</label>
+          <label for="shaken">{{localize "STATUS_EFFECT.SHAKEN"}}</label>
         </div>
       </div>
       <div class="attribute-stats enraged">
         <input type="checkbox" id="enraged" data-dtype="Boolean" name="system.condition.enraged.value" {{checked this.actor.conditions.enraged.value}} />
-        <label for="enraged"> Enraged</label>
+        <label for="enraged">{{localize "STATUS_EFFECT.ENRAGED"}}</label>
       </div>
       <div class="attribute-stats poisoned">
         <input type="checkbox" id="poisoned" data-dtype="Boolean" name="system.condition.poisoned.value" {{checked this.actor.conditions.poisoned.value}} />
-        <label for="poisoned"> Poisoned</label>
+        <label for="poisoned">{{localize "STATUS_EFFECT.POISONED"}}</label>
       </div>
     </div>
 
     <div class="traits-section">
-      <h3>Traits <i class="fas fa-fingerprint"></i> </h3>
+      <h3>{{localize "BIO.TRAITS"}}<i class="fas fa-fingerprint"></i> </h3>
       <div class="traits">
         <div class="trait-with-answer">
-          <h4>Identity:
+          <h4>{{localize "BIO.IDENTITY"}}:
```

Comment:
All punctuation, including colons should be within the localized string!

Reply:
TIL: https://softwareengineering.stackexchange.com/a/234554. Thanks! I'll make that change shortly